conda/plugins/reporter_backends/console.py

```diff
@@ -62,7 +62,7 @@ def conda_reporter_backends():
     Reporter backend for console
     """
     yield CondaReporterBackend(
-        name="console",
+        name=DEFAULT_CONSOLE_REPORTER_BACKEND,
```

Comment:
Let's use this for the strict definition of the default console reporter backed via a prioritized call order, similar how we use the `tryfirst` parameter of pluggy to prevent overriding the classic solver in https://github.com/conda/conda/blob/156a69dee64ce94f39c77271d3e558457e04fb43/conda/plugins/solvers.py#L9

That will assure hat the plugin loading method above will be able to always have a default console reporter backend to load.

```python
@hookimpl(tryfirst=True)  # make sure the default console reporter backend can't be overridden
def conda_reporter_backends():
    """
    Reporter backend for console
    """
    yield CondaReporterBackend(
        name=DEFAULT_CONSOLE_REPORTER_BACKEND,
        description="Default implementation for console reporting in conda",
        renderer=ConsoleReporterRenderer,
    )
```